ceph/ceph_admin/__init__.py

```diff
@@ -116,9 +118,14 @@ def set_tool_repo(self, repo=None):
             base_url = self.config["base_url"]
             if not base_url.endswith("/"):
                 base_url += "/"
-            base_url += "compose/Tools/x86_64/os/"
+            if cloud_type == "ibmc":
```

Comment:
IMO, I think a common definition which gives us all repos based on the infra environment would be good.

Reply:
Including for 4x, 5x and any versions.